### PR TITLE
Implement taskadd command

### PR DESCRIPTION
**Pull request by khinkhinn**
- EditTagDescriptor has been modified to only have one attribute, newTagTask instead
- Currently not integrated with UI components

### CODE REVIEW

**Review by siqiang-ng — APPROVED** [review]
LGTM

**Review by JoeyToh — COMMENTED** [review]
lgtm

File: src/main/java/seedu/address/model/tag/Tag.java [hunk 1]
```diff
@@ -43,6 +43,10 @@ public TagName getTagName() {
         return this.tagName;
     }
 
+    public List<TagTask> getModifiableTagTask() {
+        return this.tagTasks;
+    }
+
     public List<TagTask> getTagTasks() {
         return Collections.unmodifiableList(this.tagTasks);
     }
```
Comment (akgrenSoar) [inline src/main/java/seedu/address/model/tag/Tag.java]: The intention was for Tag to be immutable. (modifying fields is not allowed)
I'm not sure how `EditTagDescriptor` works though, so i'll leave it to your discretion.

**Review by akgrenSoar — APPROVED** [review]
LGTM, except for one comment.

File: src/main/java/seedu/address/model/tag/Tag.java (hunk 1 above)
Comment (khinkhinn) [reply src/main/java/seedu/address/model/tag/Tag.java]: yep i removed it thanks!!

**Review by akgrenSoar — APPROVED** [review]
LGTM

**Pull request merged**